backstop/tests/menu.html

```diff
@@ -29,162 +40,1879 @@
+            />
+          </svg>
+          <span class="iui-label">
+            <div class="iui-menu-label">Beta</div>
+            <div class="iui-menu-description">Lorem ipsum</div>
+          </span>
+          <!-- Use caret-right-small.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            viewBox="0 0 16 16"
+            aria-hidden="true"
+          >
+            <path
+              d="m6.00263 4.80681v6.39946a.27938.27938 0 0 0 .44292.24059l3.45479-3.17567a.34022.34022 0 0 0 .00046-.4807l-.00046-.00044-3.45479-3.22382a.269.269 0 0 0 -.44292.24058z"
+            />
+          </svg>
+        </li>
+        <li
+          class="iui-menu-item iui-large"
+          tabindex="0"
+        >
+          <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            viewBox="0 0 16 16"
+            aria-hidden="true"
+          >
+            <path
+              d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+            />
+          </svg>
+          <span class="iui-label">
+            <div class="iui-menu-label">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et
+              dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+              ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+              fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+              mollit anim id est laborum.</div>
+            <div class="iui-menu-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+              ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+              voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+              proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+          </span>
+          <!-- Use caret-right-small.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            viewBox="0 0 16 16"
+            aria-hidden="true"
+          >
+            <path
+              d="m6.00263 4.80681v6.39946a.27938.27938 0 0 0 .44292.24059l3.45479-3.17567a.34022.34022 0 0 0 .00046-.4807l-.00046-.00044-3.45479-3.22382a.269.269 0 0 0 -.44292.24058z"
+            />
+          </svg>
+        </li>
+        <li class="iui-menu-divider"></li>
```

Comment:
Add `aria-hidden` to dividers

Reply:
`aria-hidden="true"` added to all `.iui-menu-divider`s.